components/automate-ui/src/app/modules/user/user-details/user-details.component.ts

```diff
@@ -34,76 +41,51 @@ export type UserTabName = 'password' | 'details';
   styleUrls: ['./user-details.component.scss']
 })
 export class UserDetailsComponent implements OnInit, OnDestroy {
-  public isAdminView = false;
-  public user: User;
-  public displayNameForm: FormGroup;
-  public passwordForm: FormGroup;
   private isDestroyed = new Subject<boolean>();
+  public loading = true;
   public tabValue: UserTabName = 'details';
   private url: string;
-  public saveSuccessful = false;
-  public saveInProgress = false;
-  public loadingGetUser$: Observable<boolean>;
+  public userDetails: UserDetails;
```

Comment:
The UserDetails represents three cases: Admin user editing other users, admin user editing their own user, and profile page.